returnn/util/file_cache.py

```diff
@@ -381,6 +392,7 @@ def __init__(self, *, interval: float = 1.0, cache_base_dir: str):
         self.files = defaultdict(int)  # usage counter
         self.interval = interval
         self.cache_base_dir = cache_base_dir
+        self._started = False
```

Comment:
This is an attrib you are overwriting from the underlying `Thread` object. See failing test. I think the underlying `Thread` actually already does exactly this logic?

Reply:
ohhh damn! the docs say there will be a runtimeerror when you invoke start multiple times

Reply:
```
if self._started.is_set():
    raise RuntimeError("threads can only be started once")
```